src/main/public/js/double-click-preventer.js

```diff
@@ -0,0 +1,17 @@
+(function () {
+  document.addEventListener('DOMContentLoaded', function (event) {
```

Comment:
Remove unused parameter.

Reply:
Removed.

Reply:
Not removed. Only 50% of what I asked got removed.

Reply:
Ohh yes, sorry!. Removed